### PR TITLE
Added the ISP freedom.nl to the autoconfig database.

### PR DESCRIPTION
**Pull request by jhaand**
Hi,

I added the ISP freedom.nl to the database. Is there a way I can test this with Thunderbird? 

Also mentioned in: https://bugzilla.mozilla.org/show_bug.cgi?id=1808178

best regards,
Jelle

### CODE REVIEW

File: ispdb/freedom.nl.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    </incomingServer>
+    <outgoingServer type="smtp">
+      <hostname>smtp.freedom.nl</hostname>
+      <port>465</port>
+      <socketType>SSL</socketType>
+      <username>%EMAILADDRESS%</username>
+      <authentication>password-cleartext</authentication>
+    </outgoingServer>
+  </emailProvider>
+  <documentation url="https://helpdesk.freedom.nl/category-detail/hoe-stel-ik-mijn-e-mail-in-op-pc-en-laptop">
```
Comment (cketti) [inline ispdb/freedom.nl.xml:21]: The `documentation` element should be a child of the `emailProvider` element, not a sibling.

**Comment by cketti** [comment]
I've contacted the email provider to ask them to host the config file themselves. I'll post another comment if I receive a response.

**Comment by jhaand** [comment]
OK. Thank you for looking. \
I moved the `documentation` into the `emailProvider` element. 
It would of course really help if every provider would maintain their own entry.

**Comment by jhaand** [comment]
Any news from Freedom.nl about hosting the file themselves?

**Comment by lucasvbeek** [comment]
@jhaand @cketti I've added the autoconfiguration file at [https://freedom.nl/.well-known/autoconfig/mail/config-v1.1.xml](https://freedom.nl/.well-known/autoconfig/mail/config-v1.1.xml). Thank you for bringing this to our attention!

**Comment by cketti** [comment]
@lucasvbeek: Awesome. Thank you :+1:

**Pull request closed without merging**